src/float.rkt

```diff
@@ -24,7 +25,22 @@
      (+ (ulp-difference (real-part x) (real-part y))
         (ulp-difference (imag-part x) (imag-part y)))]
     [((? boolean?) (? boolean?))
-     (if (equal? x y) 0 64)]))
+     (if (equal? x y) 0 64)]
+    [((? posit8?) (? posit8?))
+     (p8s-between x y)]
+    [((? posit16?) (? posit16?))
+     (p16s-between x y)]
+    [((? posit32?) (? posit32?))
+     (p32s-between x y)]
+    [((? quire8?) (? quire8?))
+     (ulp-difference (posit8->double (quire8->posit8 x))
+                     (posit8->double (quire8->posit8 y)))]
```

Comment:
This should be changed to use `p8s-between`, right? In any case, worth thinking for the future about error measures in mixed-type computations. We'd prefer to compare quires without truncating them!